sys/include/net/loramac.h

```diff
@@ -304,6 +304,15 @@ extern "C" {
  * @name    LoRaMAC parameters indexes
  */
 
+/**
+ * @brief   Device class
+ */
+enum {
+    LORAMAC_CLASS_A,                   /**< Class A device */
+    LORAMAC_CLASS_B,                   /**< Class B device */
+    LORAMAC_CLASS_C,                   /**< Class C device */
+};
```

Comment:
`typedef ... loramac_class_t;`?

Reply:
why not

Reply:
well there are other enums in this file not using `typedef`. Should I update them as well ?

Reply:
Not in this PR, but if *it makes sense*, why not?